example/src/index.css

```diff
@@ -16,6 +16,61 @@ label {
   margin-bottom: 32px;
 }
 
+/*CONTAINER - typograph.jsx*/
+.container > h1{
```

Comment:
Horen thuis in typography.scss. Deze file is enkel voor styling die van toepassing is op Catalog (zoals het mooi/duidelijk weergeven van componentjes in de "show" website). Stijlen die hier in index.css worden toegevoegd zijn enkel in Catalog beschikbaar.

 H1 - H6 etc moeten ook beschikbaar worden in de June-app.